client/src/CodeClaimPage/ClaimForm.tsx

```diff
@@ -80,6 +101,7 @@ const ClaimForm: React.FC<{
                       <input
                         type="text"
                         autoComplete="off"
+                        style={{ borderColor: mainColor ? mainColor : COLORS.primaryColor }}
```

Comment:
Prefer nullish coalescing operator

```suggestion
                        style={{ borderColor: mainColor ?? COLORS.primaryColor }}
```